signal/sigsnoop.bt

```diff
@@ -0,0 +1,103 @@
+#!/bin/env bpftrace
+/**
+ * sigsnoop.bt - Trace standard and real-time signals.
+ *               For Linux, uses bpftrace and eBPF.
+ *
+ * This is a bpftrace version of the bcc libbpf-tools/sigsnoop of the same name.
+ *
+ * USAGE: sigsnoop.bt
+ *
+ * Copyright (C) 2024 CESTC, Co.
+ *
+ * 08-Apr-2024	Rong Tao	Create this.
```

Comment:
We're also not going to maintain a pseudo-changelog in the script headers in this repository as it's not a reliable source of information (some people update it, some don't) and to avoid debates on what changes are worth including. The Git logs should provide a complete history.

The copyright notice will of course remain.

```suggestion
```